.github/pull_request_template.md

```diff
@@ -11,16 +11,15 @@ Ready for review / Work in progress
 
 * Description: 
 
-* Fixes Issue#<issue-number>
+* Fixes [Issue#<issue-number>](url)
```

Comment:
Let's keep this simple, no need to add markdown formatting. In fact, note that GitHub auto-closes issues once the associated PR is merged if it follows the format:

`Fixes #123` (where 123 is the issue number)

So I would change this in the template to:

`Fixes Issue#`